### PR TITLE
Update versions in ci workflow

### PR DESCRIPTION
**Pull request by reugn**

### CODE REVIEW

**Review by pbaerospike — APPROVED** [review]

**Comment by pbaerospike** [comment]
Sorry - I should have caught the CI stuff.  ;)

**Review by roimenashe — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -16,11 +16,11 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@v1
         with:
-          go-version: '1.14.9'
+          go-version: '1.17.5'
```
Comment (jgiannuzzi) [inline .github/workflows/ci.yml:19]: As Go 1.17.6 has been released in the meantime, we might as well upgrade to it :)
```suggestion
          go-version: '1.17.6'
```

**Review by jgiannuzzi — APPROVED** [review]
Tested locally with these versions of Goreleaser and Go - LGTM 👍

**Comment by jgiannuzzi** [comment]
Thanks for the new release @reugn 🚀

**Pull request merged**